lib/resty/core/request.lua

```diff
@@ -389,14 +389,7 @@ do
     _M.set_req_header = set_req_header
 
 
-    local orig_func = ngx.req.set_header
```

Comment:
Does this mean we can remove the non-FFI implementation of `ngx.req.set_header`? I haven't checked, just asking out loud.

Reply:
@thibaultcha 
Yes.